public/js/components/Scopes.js

```diff
@@ -3,29 +3,30 @@
 const React = require("react");
 const { bindActionCreators } = require("redux");
 const { connect } = require("react-redux");
-const classnames = require("classnames");
-const { fromJS, Map } = require("immutable");
 const ImPropTypes = require("react-immutable-proptypes");
 const actions = require("../actions");
 const { getPause, getLoadedObjects } = require("../selectors");
+const ObjectInspector = React.createFactory(require("./ObjectInspector"));
 const { DOM: dom, PropTypes } = React;
-const ManagedTree = React.createFactory(require("./util/ManagedTree"));
-const Arrow = React.createFactory(require("./util/Arrow"));
-const Rep = require("./Rep");
 
 require("./Scopes.css");
 
 function info(text) {
   return dom.div({ className: "pane-info" }, text);
 }
 
-function getBindingVariables(bindings) {
+function getBindingVariables(bindings, parentName) {
```

Comment:
A little bit of docs would be helpful here. Not sure what bindings refer to exactly.